src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -787,17 +787,18 @@ static UCS_CLASS_INIT_FUNC(uct_gga_mlx5_iface_t,
     uct_ib_mlx5_md_t *md                = ucs_derived_of(tl_md, uct_ib_mlx5_md_t);
     uct_ib_iface_init_attr_t init_attr  = {};
     ucs_status_t status;
+    uct_ib_mlx5_dp_ordering_t dp_ordering;
 
     init_attr.qp_type               = IBV_QPT_RC;
     init_attr.cq_len[UCT_IB_DIR_TX] = config->super.tx_cq_len;
     init_attr.max_rd_atomic         = IBV_DEV_ATTR(&md->super.dev,
                                                    max_qp_rd_atom);
     init_attr.tx_moderation         = config->super.tx_cq_moderation;
+    dp_ordering                     = ucs_min(md->dp_ordering_cap_devx.rc,
```

Comment:
Maybe revert? Seems the only change is `dp_ordering_cap` -> `dp_ordering_cap_devx`..